apps/algorithm/search/src/server.py

```diff
@@ -15,6 +15,18 @@
 es_endpoint = os.getenv("ES_ENDPOINT")
 es = Elasticsearch([es_endpoint], verify_certs=False)
 
+if not es.indices.exists(index=DEFAULT_INDEX):
+    es.indices.create(
+        index=DEFAULT_INDEX,
+        body={
+            "mappings": {
+                "properties": {
+                    "location": {"type": "geo_point"},
+                }
+            }
+        },
+    )
+
```

Comment:
Top level code is usually scary, usually only constants are up here. I guess having it here makes sense, but it still creeps me out a little